README.md

```diff
@@ -29,7 +29,7 @@ To build jQuery, you need to have the latest Node.js/npm and git 1.7 or later. E
 
 For Windows, you have to download and install [git](https://git-scm.com/downloads) and [Node.js](https://nodejs.org/en/download/).
 
-OS X users should install [Homebrew](https://brew.sh/). Once Homebrew is installed, run `brew install git` to install git,
+MacOS users should install [Homebrew](https://brew.sh/). Once Homebrew is installed, run `brew install git` to install git,
```

Comment:
AFAIK it's macOS, though, not MacOS.

Reply:
thanks. Yep, I realised that as I made the PR... I'll push a commit to update